### ISSUE TITLE
Finetuning does not work for binary classification

### ISSUE THREAD

**Issue by arushk1** [facebook/fb.resnet.torch#19]
Hello, I'm trying to finetune the resnet for binary classification, but the validation accuracy is remains constant around 50. The loss is also not decreasing. 

Any help?

**Comment by ouceduxzk** [comment]
try small LR
